src/schema/v2/show.ts

```diff
@@ -56,6 +57,18 @@ const FollowArtistType = new GraphQLObjectType<any, ResolverContext>({
   }),
 })
 
+export const ExhibitionPeriodFormatType = new GraphQLEnumType({
+  name: "ExhibitionPeriodFormat",
+  values: {
+    SHORT: {
+      value: "SHORT",
+    },
+    LONG: {
+      value: "LONG",
```

Comment:
**suggestion**: We're a bit inconsistent here but unless the backend needs the value to the uppercase, which is sometimes need to support sorting via Gravity, I think most other enums have lowercase values.

```typescript
{
  SHORT: { value: "short" },
  LONG: { value: "long" }
}
```